vm/devices/tpm/src/tpm_helper.rs

```diff
@@ -606,15 +606,25 @@ impl TpmEngineHelper {
                                 }
                             })?;
 
-                            self.nv_write(TPM20_RH_OWNER, None, TPM_NV_INDEX_AIK_CERT, &cert)
-                                .map_err(|error| TpmHelperError::TpmCommandError {
-                                    command_debug_info: CommandDebugInfo {
-                                        command_code: CommandCodeEnum::NV_Write,
-                                        auth_handle: Some(TPM20_RH_OWNER),
-                                        nv_index: Some(TPM_NV_INDEX_AIK_CERT),
-                                    },
-                                    error,
-                                })?;
+                            let res =
```

Comment:
do you want to use match so that we can use info! on the successful path and error! on the failing path, be consistent with what you did above? And in that case success might also not be necessary.